extensions/notebook/src/extension.ts

```diff
@@ -47,6 +47,7 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
 	extensionContext.subscriptions.push(vscode.commands.registerCommand('notebook.command.openBook', () => bookTreeViewProvider.openNewBook()));
 	extensionContext.subscriptions.push(vscode.commands.registerCommand('notebook.command.closeBook', (book: any) => bookTreeViewProvider.closeBook(book)));
 	extensionContext.subscriptions.push(vscode.commands.registerCommand('notebook.command.closeNotebook', (book: any) => bookTreeViewProvider.closeBook(book)));
+	extensionContext.subscriptions.push(vscode.commands.registerCommand('notebook.command.removeNotebook', (book: any) => bookTreeViewProvider.removeNotebook(book)));
```

Comment:
Generally it's good to give the params the types that you expect. So BookTreeItem in this case. 

(granted anyone can call this at any time and pass in whatever, but that isn't something I generally see value in defending against. But at the very least informing people looking at the code as to what you expect it to be is useful)